<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mbServerDialogFindReplace</class>
 <widget class="QDialog" name="mbServerDialogFindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find/Replace</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbFind">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblReplace">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbReplace">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chbMatchCase">
         <property name="text">
          <string>Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbMatchWord">
         <property name="text">
          <string>Match whole word only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbEscapeSequence">
         <property name="text">
          <string>Escape sequence</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnFindNext">
       <property name="text">
        <string>Find Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFindPrevious">
       <property name="text">
        <string>Find Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplaceAll">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
